### ISSUE TITLE
BUG REPORT (ALERT FROM CONSOLE)

### ISSUE THREAD

**Issue by soyatlas** [bmaltais/kohya_ss#989]
Hello all, 
1 - sorry my bad english.
2 - sorry my ignorance.
3 - I wish comunicate this error showed in console running th fantastic "Konya_ss" THANKS !.
4 - I don´t know if this is the correct place.
5 - the cut and paste text:

I:\kohya_ss\venv\lib\site-packages\accelerate\accelerator.py:258: FutureWarning: `logging_dir` is deprecated and will be removed in version 0.18.0 of 🤗 Accelerate. Use `project_dir` instead.
  warnings.warn(
Using accelerator 0.15.0 or above.
loading model for process 0/1
load StableDiffusion checkpoint: I:/stable-diffusion-webui/models/Stable-diffusion/OFICIAL/v1-5-pruned.ckpt
loading u-net: <All keys matched successfully>
loading vae: <All keys matched successfully>
loading text encoder: <All keys matched successfully>
CrossAttention.forward has been replaced to enable xformers.
import network module: networks.lora
[Dataset 0]
caching latents.
 40%|████████████████████████████████                                                | 1008/2511 [00:44<01:01, 24.47it/s]09:53:30-181978 INFO     Save...
100%|████████████████████████████████████████████████████████████████████████████████| 2511/2511 [01:46<00:00, 23.58it/s]
create LoRA network. base dim (rank): 8, alpha: 1.0
neuron dropout: p=None, rank dropout: p=None, module dropout: p=None
create LoRA for Text Encoder: 72 modules.
create LoRA for U-Net: 192 modules.
enable LoRA for text encoder
enable LoRA for U-Net
preparing optimizer, data loader etc.

===================================BUG REPORT===================================
Welcome to bitsandbytes. For bug reports, please submit your error trace to: https://github.com/TimDettmers/bitsandbytes/issues
For effortless bug reporting copy-paste your error into this form: https://docs.google.com/forms/d/e/1FAIpQLScPB8emS3Thkp66nvqwmjTEgxp8Y9ufuWTzFyr9kJ5AoI47dQ/viewform?usp=sf_link
================================================================================
CUDA SETUP: Loading binary I:\kohya_ss\venv\lib\site-packages\bitsandbytes\libbitsandbytes_cuda116.dll...
use 8-bit AdamW optimizer | {}
running training / 学習開始
  num train images * repeats / 学習画像の数×繰り返し回数: 6260
  num reg images / 正則化画像の数: 2400
  num batches per epoch / 1epochのバッチ数: 6260
  num epochs / epoch数: 10
  batch size per device / バッチサイズ: 2
  gradient accumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 62600
steps:   0%|                                                                                   | 0/62600 [00:00<?, ?it/s]
epoch 1/10
I:\kohya_ss\venv\lib\site-packages\xformers\ops\fmha\flash.py:339: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()
  and inp.query.storage().data_ptr() == inp.key.storage().data_ptr()
steps:   3%|█▍                                                        | 1613/62600 [05:46<3:38:17,  4.66it/s, loss=0.115steps:   3%|█▍                                                        | 1614/62600 [05:46<3:38:17,  4.66it/s, loss=0.115steps:   3%|█▍                                                        | 1614/62600 [05:46<3:38:17,  4.66it/s, loss=0.115steps:   3%|█▍                                                        | 1615/62600 [05:46<3:38:16,  4.66it/s, loss=0.115stepstepsteps:   3%|█▋

---

I had sended to the google form too.
Thanks to all !
